### ISSUE TITLE
Produce click always when an additional one finger touches and quickly untouches

### ISSUE THREAD

**Issue by dumblob** [atareao/Touchpad-Indicator#93]
First of all, I would like to thank you for this piece of SW. Without this I would not be able to use my new laptop.

Now back to my feature request :smile:.

Imagine a situation when one or more fingers are already touching the touchpad at the same time (typically during/after scrolling or multifinger gestures).

Then after stopping all those fingers stay on the touchpad.

And now if I wanted to click, I need to lift all the fingers (which are resting on the touchpad) and only then I can do the "touch & untouch" to produce the click.

My request here is to actually allow for the "touch & untouch" combo **without** the need to lift the fingers which already touch the touchpad.

This I envision to work for LMB click, RMB click, as well as for MMB click (e.g. assuming touchpad supporting up to 4 fingers, I can scroll using two fingers on my right hand and then stop scrolling, do NOT lift the two fingers but instead "touch & untouch" with two fingers of my left hand to achieve left-mouse-button click).

In other words, any touching fingers are considered as "new baseline for clicks" at any given point in time.

Of course if the "new baseline" could me made to work not just for clicks but also for drags/movements, that would become nirvana, but for my use case the support for "new baseline for clicks" (not drags/movements) is sufficient :wink:

Do you think this is doable?

**Comment by dumblob** [comment]
@atareao would you accept patches? Or is this something out of scope of this project? If so, would you have any alternative suggestions?

**Comment by atareao** [comment]
Please, do it !!!

**Comment by dumblob** [comment]
Thanks! Just to manage expectations - this might take me (many) months, so anybody interested feel free to reach out to me to join efforts :wink:.
